common/cont-lib/usr/share/cont-docs/70-general.txt

```diff
@@ -0,0 +1,16 @@
+General container help
+----------------------
+
+Run `docker run THIS_IMAGE container-usage` to get this help.
+
+Run `docker run -ti THIS_IMAGE bash` to obtain interactive shell.
+
+Run `docker exec -ti CONTAINERID container-entrypoint` to access already running container.
```

Comment:
@mfojtik wrote:
> we still don't like the container-entrypoint helper as enabler for SCL
> collections....

Its not ideal, yes.  But doing

    `docker exec CONTID container-entrypoint your command`

... instead of ...

    `docker exec CONTID your command`

does not seem to be _that_ bad.  We could consider this to be a temporary
workaround (untill Docker allows us to do that better way).

Pavel